client/src/js/components/bhNavigation.js

```diff
@@ -147,6 +147,10 @@ function NavigationController($location, $rootScope, Tree, AppCache, Notify, $tr
     units.forEach(unit => {
       unitsIndex.id[unit.id] = unit;
       unitsIndex.path[unit.path] = unit;
+
+      // See if it is a settings page and needs a "gear" icon
+      unit.settings = unit.name.indexOf('[SETTINGS]') >= 0;
```

Comment:
This is a boolean, right?  I propose that you name is `isSettings` or `isSettingsNode` or something like that.  Typically we name boolean variables with `is` or `has` or `should` or something like that.

Reply:
Updated